server.js

```diff
@@ -13,7 +12,6 @@ const OAuthRoutes = require("./routes/oauth");
 const UserRoutes = require("./routes/user");
 
 const app = express();
-app.use(bodyParser.json());
```

Comment:
I took out the global `json` body parser so that the `scan` route could use the `urlencoded` body parser.